core/src/main/scala/precog/SbtPrecogBase.scala

```diff
@@ -257,20 +250,24 @@ abstract class SbtPrecogBase extends AutoPlugin {
         List(
           WorkflowStep.Checkout,
           WorkflowStep.SetupScala,
-          WorkflowStep.Use(
-            "actions",
-            "setup-ruby",
-            "v1",
-            name = Some("Install Ruby"),
-            params = Map("ruby-version" -> "2.6")),
           WorkflowStep.Sbt(
             List("transferCommonResources", "exportSecretsForActions"),
             name = Some("Common sbt setup")),
-          WorkflowStep.Run(List("./scripts/commonSetup")),
-          WorkflowStep.ComputeVar("CLONE_DIR", "mktemp -d /tmp/precog-bump.XXXXXXXX"),
-          WorkflowStep.ComputeVar("PR_NUMBER", s"echo $$GITHUB_REF | cut -d'/' -f3"),
-          WorkflowStep.Run(List("./scripts/checkAndAutoMerge"))),
-        cond = Some("github.event_name == 'pull_request' && contains(github.head_ref, 'version-bump')"),
+
+          WorkflowStep.Run(
+            List(
+              "curl -L https://github.com/precog/devtools/raw/master/bin/sdmerge > /tmp/sdmerge",
```

Comment:
@djspiewak are we ok with not fixing the version of `sdmerge` for reproducibility here?
I guess `sdmerge` doesn't change very often.

Reply:
Yeah I think it's fine. We were already using `HEAD` implicitly before.